content/working_with/console/widgets/blueprints.md

```diff
@@ -53,6 +53,48 @@ There are also action buttons to upload a blueprint, create deployment, delete b
 The default visibility is "Tenant", and according to the logged-in user's permissions you can also choose other levels of [resource visibilities]({{< relref "working_with/manager/resource-visibility.md" >}}).<br>
 6. Click **Upload**.
 
+##### Using Terraform module
+1. Click the **Upload** button.
+2. In the menu, click **Upload from Terraform module** option.
+3. **Create blueprint from Terraform** modal will appear
+4. Enter the `Blueprint name`   
+   `Blueprint name` is the name with which you want to identify this blueprint once uploaded
+5. Select `Terraform version` (as a default, one of the terraform versions will be selected)
+6. Provide `Blueprint main information`, which consists of:
+   - `URL to a zip archive that contains the Terraform module`,
+   - `Terraform folder in the archive` - it's a selectable field from which you'll be able to select directories found inside the linked zip file.<br />
+   By default this field is disabled, until the `URL to a zip archive that contains the Terraform module` is being provided.
+   - (Optional) Credentials required for accessing `URL to a zip archive that contains the Terraform module`.
+7. (Optional) Add `Variables`, by:   
+   1. Clicking the `Variables` dropdown
+   2. Clicking the **Add** button
+   3. Filling the row fields:
+      - `Variable` - name of the variable
+      - `Source` - type of the variable, which may be selected from a dropdown<br />
+         Currently available values are: `Secret`, `Input`, `Static`.
+      - `Value / Secret key / Input name` - value of the variable (related to the selected `Source`)<br />
+         If we have not selected `Secret` `Source`, then we are able to type our own values.<br />
+         Upon selecting the `Secret` `Source` we are getting the ability to select one of the stored secrets.
```

Comment:
I think a bit more clean would be:
```suggestion
         If we have not selected `Secret` as a `Source`, then we are able to type our own values.<br />
         Upon selecting the `Secret` as a `Source` we are getting the ability to select one of the stored secrets.
```

but maybe it would be even better to describe each individual source types:
- by selecting `Secret` as a `Source`, we are able to...
- by selecting `Input` as a `Source`, we are able to...
- by selecting `Static` as a `Source`, we are able to...

Reply:
I agree, but I also think we should prefer a person-less form: 'selecting `Input` as a `Source` enables to...'. If not, we should at least avoid plural form and use 'you' instead of 'we'.

Reply:
Nicely looking idea!
I've added those modifications accordingly 😉